osi_vehicle.proto

```diff
@@ -0,0 +1,692 @@
+        // 
+        // Negative camber if the bottom of the wheel is farther out than the
+        // top.
+        // For more information: https://en.wikipedia.org/wiki/Camber_angle.
+        //
+        // Unit: [rad]
+        //
+        optional double camber = 5;
+
+        // Contains the tirepressure of each tire.
+        //
+        // Unit: [Pa]
+        // 
+        optional double tirepressure = 6;
+
+        // Contains the springdeflection in z-direction for each wheel.
+        //
+        // Unit: [mm]
+        //
+        optional double springdeflection = 7;
+
+        //Contains the relativ position of the wheel to the center of the car
+        //
+        //Unit: [m]
+        //
+        optional Vector3d position = 8;
+    
+        //Contains the relativ orientation of the wheel to the center of the car
+        //As the rotation of the wheel is also controlled by this value, 
+        //
+        //Unit: [rad]
+        //
+        optional Orientation3d orientation = 9;
+
+        //Contains the absolute (longitunal) slip of the tire
+        //0-100 percent
+        //
+        //Unit: []
+        //
+        optional double slip = 10;
+
+        //Contains the slip-angle of the tire
+        //
+        //Unit: [rad]
+        //
+        optional double slipangle = 11;
+    }
+
+    //
+    // \brief Information about the automated driving system.
+    // Important is the differentiation between states and intentions.
+    // An automated-driving function itself does not have many states. It is more about Intentions,
+    // to change some vehicle states. E.g. a desired acceleration(-pedal-position), but effective is the one in
+    // the powertrain message.
+    //
+    message VehicleAutomatedDrivingFunction
+    {
+        //
+        // Here the name of the function can be chosen.
+        // E.g.: Hadpilot, longitudinal_control, lateral_control or emergency_brake_assistant.
+        //
+        optional string function_name = 1;
+
+        //
+        // States of an ad-function.
+        //
+        optional States states = 2;
+        
+        // 
+        // Intentions, requests or wishes from the function to change the vehicle-state.
+        //
+        optional Intentions intentions = 3;
+
+        //
+        // \brief Internal states of an automated-driving function.  
+        //    
+        //
+        message States
+        {
+            // Activation state of the function.
+            //
+            optional bool is_activated = 1;
+
+            // This is the speed the function targets.
+            // E.g.: At the point of activation, the actual speed could be 80 km/h, 
+            // but the function tries to accelerate to 130 km/h.
+            //
+            // Unit: [km/h]
+            //
+            optional double targeted_speed = 2;
+
+            // The timegap describes the minimumdistance to the next vehicle in front.
+            //
+            // Unit: [s]
```

Comment:
The minimum distance in seconds? This is also a strange definition

Reply:
E.g. ACC works with a distance in seconds, because meters depends on the velocity you drive